packages/block-library/src/latest-comments/edit.js

```diff
@@ -31,22 +33,51 @@ const MIN_COMMENTS = 1;
  */
 const MAX_COMMENTS = 100;
 
-export default function LatestComments( { attributes, setAttributes } ) {
+export default function LatestComments( { attributes, setAttributes, name } ) {
 	const { commentsToShow, displayAvatar, displayDate, displayContent } =
 		attributes;
 
-	const serverSideAttributes = {
-		...attributes,
-		style: {
-			...attributes?.style,
-			spacing: undefined,
+	const dropdownMenuProps = useToolsPanelDropdownMenuProps();
+
+	const { content, status, error } = useServerSideRender( {
+		attributes,
```

Comment:
```suggestion
		attributes: serverSideAttributes,
```

Should'nt be this? why to remove `serverSideAttributes` variable?

Reply:
Hi @hbhalodia , I initially thought about removing the serverSideAttributes variable since HtmlRenderer merges both the server-rendered attributes and wrapperProps, and during my testing it didn't seem to cause any visual issues. I've reverted changes to keep serverSideAttributes.